src/controllers/routers/beneficiaries.js

```diff
@@ -126,6 +129,45 @@ const getBeneficiary = async (req, res, next) => {
   }
 };
 
+/**
+ * Retrieve related cases by id
+ * @param {Request} req
+ * @param {Response} res
+ */
+const getBeneficiaryCases = async (req, res, next) => {
```

Comment:
since the table we're accessing is cases, move this to the cases controller file.
the route is okay. the route should be in beneficiaries.

rename to getCasesByBeneficiaryId

Reply:
moved getCasesByBeneficiaryId from controllers > beneficiary to controllers > cases
update routers > beneficiary to access cases.getCasesByBeneficiaryId